### PR TITLE
Feature/fix init

### PR DESCRIPTION
**Pull request by faceleg**
This plugin would not initialise for me, giving the following error:

```TEXT
Seneca Fatal Error
==================

Message: seneca: Action hook:init,role:mail,sub:templates responded with result that was not an object or array: [Function: _template]; Use option strict:{result:false} to allow; arguments were: { role: 'mail',  hook: 'init',  sub: 'templates',  options:    { content: {},     mail: { from: 'contact@66pix.com' },     transport: 'smtp',     config: { port: '465', host: 'host400.hostmonster.com', auth: [Object] },     folder: './templates/email/views' } }

Code: result_not_objarr

Details: { pattern: 'hook:init,role:mail,sub:templates',
  args: '{ role: \'mail\',  hook: \'init\',  sub: \'templates\',  options:    { content: {},     mail: { from: \'contact@66pix.com\' },     transport: \'smtp\',     config: { port: \'465\', host: \'host400.hostmonster.com\', auth: [Object] },     folder: \'./templates/email/views\' } }',
  result: [Function: _template],
  plugin: {} }
```

I changed the initTemplates and initTransport actions to callback with an empty object.

### CODE REVIEW

**Comment by faceleg** [comment]
Bump!

File: lib/templateActions.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ module.exports = function (seneca, options) {
         }
 
         template = templateinstance
-        callback(null, template)
+        callback(null, {})
```
Comment (guyellis) [inline lib/templateActions.js]: Is this the right fix?
Should the line above perhaps read something like this instead:
`template = _.isObject(templateinstance) || _.isArray(templateinstance) ? templateinstance : {};`

cc/ @rjrodger

Comment (faceleg) [inline lib/templateActions.js]: I'm not sure if `templateinstance` will ever be anything but a function

**Comment by faceleg** [comment]
Sorry, bumping this again

**Comment by rjrodger** [comment]
@mirceaalexandru over to you!

**Comment by mirceaalexandru** [comment]
Thank you Michael for your fix, sorry it took so long.

**Comment by faceleg** [comment]
No problem, thanks for merging & doing a release :)

**Pull request merged**